Azure/sonic-mgmt
# New T0 Test case to check connectivity of statically added arp entries

### Description of PR

Summary:
Fixes # (Issue)

### Type of change



- [x ] Test case(new/improvement)


### Back port request

### Approach
#### What is the motivation for this PR?
This is a new T0 test case that addresses the following gap in coverage:
There was an issue where devices connected to ports part of vlan on the DUT could not be reached from outside the VLAN subnet after static ARP/neighbor entries created and deleted multiple times for devices. 
There was no community test to catch this issue, hence adding it

#### How did you do it?
Setup:
- Get a VLAN ID of a vlan with at least 2 members, ipv4 and ipv6 of the vlan
- Randomized selection of 2 vlan members from the vlan and get its port index
- Generate 2 IPv4 and IPv6 addresses in the vlan's network using the port indices
- Add above IPs to ptf hosts matching the port index
- Randomized selection of neighbor VM host  from the neighbor list
- Get VM ip details and add route on PTF

Test Procedure:
- Add static arp entry of ipv4 of PTF host 1 and 2 on DUT
- Add static neighbor entry of ipv6 of PTF host 1 and 2  on DUT
- Check for connectivity from VM host
- Delete static neighbor entry of ipv6 of PTF host 1 and 2  on DUT
- Add static neighbor entry of ipv6 of PTF host 2 and then 1 on DUT
- Delete and re-add static arp entry of ipv4 of PTF host 1 and 2 on DUT
- Check for connectivity from VM host

#### How did you verify/test it?
Ran test with t0-52 topology 

#### Any platform specific information?

#### Supported testbed topology if it's a new test case?
t0
### Documentation
